configuration/config-txt/video.md

```diff
@@ -80,31 +51,31 @@ hdmi_edid_filename:1=FileForPortOne.edid
 
 ### hdmi_force_edid_audio
 
-Setting `hdmi_force_edid_audio` to `1` pretends that all audio formats are supported by the display, allowing passthrough of DTS/AC3 even when this is not reported as supported.
+Setting `hdmi_force_edid_audio` to `1` enables all audio formats, ignoring the list of supported formats reported by the display in the EDID. This is useful for allowing passthrough of DTS/AC3 even when the EDID does not indicate support for this.
 
 ### hdmi_ignore_edid_audio
 
-Setting `hdmi_ignore_edid_audio` to `1` pretends that all audio formats are unsupported by the display. This means ALSA will default to the analogue audio (headphone) jack.
+Setting `hdmi_ignore_edid_audio` to `1` disables all audio formats, ignoring the list of supported formats reported by the display in the EDID. This means Linux will default to using the analogue audio output instead.
 
 ### hdmi_force_edid_3d
 
-Setting `hdmi_force_edid_3d` to `1` pretends that all CEA modes support 3D, even when the EDID does not indicate support for this.
+Setting `hdmi_force_edid_3d` to `1` enabled support for 3D for all CEA modes that support 3D, even when the EDID does not indicate support for this.
```

Comment:
enabled -> enables ?
Although I'm not sure if the wording of "enables support for 3D for all CEA modes that support 3D" sounds right? I've got no knowledge about EDID, CEA and 3D, but the previous text seemed to imply ""enables 3D support for all CEA modes" ? :shrug:

Reply:
Yes, I changed the meaning slightly in my edit - fixed.